wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -218,6 +244,11 @@ public TypeName typeName(ProtoType protoType) {
     return candidate;
   }
 
+  public TypeName protoFieldsTypeName(ProtoType protoType) {
+    TypeName candidate = nameToProtoFieldsJavaName.get(protoType);
```

Comment:
you can probably omit the variable declaration and just return the immediate value?